socorro/external/es/connection_context.py

```diff
@@ -3,94 +3,29 @@
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import contextlib
-import datetime
-import re
 
-from configman import Namespace, RequiredConfig
-from configman.converters import list_converter
 import elasticsearch
 
-from socorro.external.es.super_search_fields import build_mapping
-from socorro.lib.libdatetime import utc_now
 
-
-# Elasticsearch indices configuration.
-ES_CUSTOM_ANALYZERS = {
-    "analyzer": {"semicolon_keywords": {"type": "pattern", "pattern": ";"}}
-}
-ES_QUERY_SETTINGS = {"default_field": "signature"}
```

Comment:
I reduced the role of the connection context to _just_ the connection things and moved all the crashstorage/search related things to the crashstorage class.